queries/rust/highlights.scm

```diff
@@ -97,7 +97,7 @@
 
 (lifetime (identifier) @label)
 
-(self) @variable.builtin
+(self) @keyword
```

Comment:
i believe self should stay a builtin variable. If you really want this to be a keyword, you could override it in one of your own query files :)

Reply:
Or link variable.builtin to keyword. It's mostly `this` and `self`

Reply:
Fair enough. I opened #327 in this regard.